backend/src/test/java/com/github/devbk84/backend/controller/ProductControllerTest.java

```diff
@@ -39,4 +45,15 @@ void getAllProducts() throws Exception {
                 .andExpect(content().json("[]"));
     }
 
+    @Test
+    @DirtiesContext
+    void getProductsById() throws Exception {
+        productRepo.save(new Product("69",
+                "Milch", "Test",
+                "orderFav", new BigDecimal(0)));
+        mockMvc.perform(get("/api/products/69"))
+                .andExpect(status().isOk());
```

Comment:
For later it is easier to seperate the product in a own Variable, so you can easily modify the test later :)
```suggestion
        Product product = new Product("69",
                "Milch", "Test",
                "orderFav", new BigDecimal(0));
        productRepo.save(product);
        mockMvc.perform(get("/api/products/" + product.id()))
                .andExpect(status().isOk());
```